includes/definitions/benu.yaml

```diff
@@ -0,0 +1,21 @@
+os: benu
```

Comment:
As this is called BenuOS then you should name it benuos. This will obviously need the file names changing.